src/elements/content-uploader/DroppableContentV2.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * @flow
+ * @file Droppable area containing upload item list
+ */
+
+import React from 'react';
+import classNames from 'classnames';
+import makeDroppable from '../common/droppable';
```

Comment:
can we avoid using this older component and use `react-dnd` or `react-beautiful-dnd`